### ISSUE TITLE
Accessibility: Missing keyboard focus

### ISSUE THREAD

**Issue by sjaeger24** [bycs-lp/mod_learningmap#147]
Students can navigate to locations using the keyboard, but the keyboard focus is not visible.

**Comment by sh-csg** [comment]
This will be resolved by  #157
